src/static/css/index.css

```diff
@@ -1,19 +1,19 @@
-body {
+header {
+  /* background-image: url(/static/images/intro-background-fit.svg); */
+  background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxOTIwIiBoZWlnaHQ9IjEyNTAiPjxkZWZzPjxzdHlsZT4uY2xzLTF7ZmlsbDojYThjYWJhO2ZpbGwtcnVsZTpldmVub2RkO29wYWNpdHk6LjA1fTwvc3R5bGU+PC9kZWZzPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS0yMzEgODloMjA4OHY2NTFsLTIwODggMzYwVjg5eiIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS00NzAgMjM5aDIwODh2NjUxbC0yMDg4IDM2MFYyMzl6Ii8+PHBhdGggZD0iTS0xNDQtMTloMjA4OHY2NTFMLTE0NCA5OTJWLTE5eiIgZmlsbD0iIzFhMmI0OSIgb3BhY2l0eT0iLjciIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS00NzAtODNoMjA4OHY2NTFMLTQ3MCA5MjhWLTgzeiIvPjxwYXRoIGQ9Ik0tMjMyLTQ4aDIwODh2NjUxTC0yMzIgOTYzVi00OHoiIGZpbGw9IiNmMmYyZjIiIG9wYWNpdHk9Ii4wNSIgZmlsbC1ydWxlPSJldmVub2RkIi8+PHBhdGggZD0iTS0xNDI2LTM3MUg2NjJ2NjUxbC0yMDg4IDM2MFYtMzcxeiIgb3BhY2l0eT0iLjEiIGZpbGw9IiNmMmYyZjIiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==);
+  background-position: bottom left;  
   background-repeat: no-repeat;
-  background-size: 100% 910px;
-}
-
-body.lazy {
-  /* This is a 1px solid #677486 image. */
-  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNML2n7DwAFDQJi8nQACgAAAABJRU5ErkJggg==);
-  background-size: 100% 700px;
+  background-size: 100% 100%;
```

Comment:
I think this is causing the graphic to look distorted/stretched on mobile:

![image](https://user-images.githubusercontent.com/1120896/68957541-3e4eda80-0798-11ea-8dd5-f111984cd4d7.png)

Live version for comparison:

![image](https://user-images.githubusercontent.com/1120896/68957577-4c9cf680-0798-11ea-812e-2a5be04cd98d.png)

Could you see if you can size the graphic similar to how it appears in production for mobile?

Reply:
I used `cover` for the background-size on displays lower than 600px. 

Also, as you may have noticed already, the `/static/images/home-hero.png` is now visually wrapped by the `intro-container` and from a design perspective that is good thing - having it look like it's standing on the edge of the skewed background.

Let me know what you think!